tutorials/bios-boot-tutorial/bios-boot-tutorial.py

```diff
@@ -293,9 +296,53 @@ def stepCreateTokens():
     run(args.cleos + 'push action eosio.token issue \'["eosio", "%s", "memo"]\' -p eosio' % intToCurrency(totalAllocation))
     sleep(1)
 def stepSetSystemContract():
+    # Nearly all of the protocol upgrade features introduced in v1.8 first require a special protocol 
```

Comment:
It is not "Nearly all" it is "all".

Reply:
"nearly all" is used here "https://developers.eos.io/eosio-nodeos/docs/consensus-protocol-upgrade-process"
I will correct.